app/src/main/res/layout/ongoing_story_card.xml

```diff
@@ -29,9 +29,10 @@
       <ImageView
         android:id="@+id/lesson_thumbnail"
         android:layout_width="80dp"
-        android:layout_height="44dp"
+        android:layout_height="0dp"
         android:layout_marginStart="8dp"
         android:importantForAccessibility="no"
```

Comment:
Remove this and add content description to this image, which can be the story name

Reply:
I have removed the attribute `importantForAccessibility` and added contentDescription as per instruction. I also performed accessibility test to check whether the content description is heard or not, the test successful and the content description is heard clearly when the user click on the list item.